### PR TITLE
Initial EIS Drivers

### PR DESCRIPTION
**Pull request by jessemapel**
Re-do of #462

### CODE REVIEW

**Comment by jessemapel** [comment]
Kernels are sufficiently "fake" right now that I'm not going to include any sliced kernels until we have some actual data. So just doing unit testing of specific parameters, which are basically all hard coded.

File: tests/pytests/data/eis_wac_pb/eis_wac_pb_isis3.lbl [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        TargetName       = Didymos
+        StartTime        = 2021-03-06T04:05:27.77
+        ExposureDuration = 1.0 <seconds>
+    End_Group
+
+    Group = BandBin
+        FilterName = BLU
+        Center     = 427.5 <nm>
+        Width      = 95 <nm>
+    End_Group
```
Comment (gsn9) [inline tests/pytests/data/eis_wac_pb/eis_wac_pb_isis3.lbl:34]: Are these the right values for the `BandBin`? I have different values in one of the tests within ISIS, I just wanted to double-check.
```
      Group = BandBin
        FilterName = CLEAR
        Center     = 712.5 <nm>
        Width      = 675 <nm>
      End_Group
```

**Review by gsn9 — APPROVED** [review]
Look's good to me, just the one comment.

File: tests/pytests/data/eis_wac_pb/eis_wac_pb_isis3.lbl (hunk 1 above)
Comment (jessemapel) [reply tests/pytests/data/eis_wac_pb/eis_wac_pb_isis3.lbl:34]: I manually changed the filters on the PB tests to check some of the filter-based logic. These are the values for the BLU filter from our ingestion template.

**Pull request merged**